src/cobald/daemon/config/yaml.py

```diff
@@ -23,3 +32,43 @@ def load_configuration(path, translator=Translator()):
                 what='dangling configuration keys (%s)' % ', '.join(config_data),
             )
         return translator.translate_hierarchy({'pipeline': root_pipeline})
+
+
+def yaml_constructor(factory):
+    """
+    Convert a factory function/class to a YAML constructor
+
+    :param factory:
```

Comment:
Description for factory is missing.

```suggestion
    :param factory: the factory function/class
```